internal/nostr/nostr.go

```diff
@@ -490,17 +492,17 @@ func (svc *Service) NIP47WebhookHandler(c echo.Context) error {
 
 func (svc *Service) prepareNIP47Subscription(relayUrl, walletPubkey, webhookUrl string, requestEvent RequestEvent) (Subscription) {
 	return Subscription{
-		RelayUrl:       relayUrl,
-		WebhookUrl:     webhookUrl,
-		Open:           true,
-		Authors:        &[]string{walletPubkey},
-		Kinds:          &[]int{NIP_47_RESPONSE_KIND},
-		Tags:           &nostr.TagMap{"e": []string{requestEvent.NostrId}},
-		Since:          time.Now(),
-		Limit:          1,
-		RequestEvent:   &requestEvent,
-		EventChan:      make(chan *nostr.Event, 1),
-		Uuid:           uuid.New().String(),
+		RelayUrl:     relayUrl,
+		WebhookUrl:   webhookUrl,
+		Open:         true,
+		Authors:      &[]string{walletPubkey},
+		Kinds:        &[]int{NIP_47_RESPONSE_KIND},
+		Tags:         &nostr.TagMap{"e": []string{requestEvent.NostrId}},
+		Since:        time.Now(),
+		Limit:        1,
+		RequestEvent: &requestEvent,
```

Comment:
the `requestEvent` argument in this method could be a pointer so we do not copy the whole object